pkg/stanza/operator/input/udp/udp.go

```diff
@@ -203,6 +184,52 @@ func (u *Input) goHandleMessages(ctx context.Context) {
 	}()
 }
 
+func VerifyLog(data []byte) (token []byte) {
```

Comment:
No need to export this, right? Also, we're doing. more than verifying here. Maybe call this `trimLog`, or `truncateMaxLog`, etc

Reply:
this is the same logic as the empty split function ("SplitNone" it's called in the splitter file). 
if you don't define a multiline then this function is the default split function that gets passed to the scanner split function, 
so this is not new logic, but it's maintaining the existing behavior. correct me if I'm wrong.

Reply:
My comment was in regards to the name of the function.

That said, if this is the same as another function, do we need it?

Reply:
I'll change the name.
but we need it to align with the behavior previous to this PR.
because until now we always called the scanner split function, and this was the default split function behavior (it's actually a bit different because the original 'SplitNone' had to be a splitter interface).

unless you are saying that this validation is redundant, and then I'll happily remove it.

Reply:
Thanks for clarifying. I think it reads better now.